pysmt/solvers/yices.py

```diff
@@ -216,7 +216,7 @@ def get_value(self, item):
             elif ty.is_bv_type():
                 width = ty.width
                 res = (ctypes.c_int32 * width)()
-                libyices.yices_val_get_bv(self.model, ctypes.byref(yval), res)
+                libyices.yices_get_bv_value(self.model, titem, res)
```

Comment:
Why is there this change? Is it a change in the API? Shall we add a regression test for this?